core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -3890,11 +3893,14 @@ private SqlDialect nonOrdinalDialect() {
         + "FROM `foodmart`.`product`";
     final String expectedSpark = "SELECT CAST(`product_id` AS CHAR(1))\n"
         + "FROM `foodmart`.`product`";
+    final String expectedClickHouse = "SELECT CAST(`product_id` AS `FixedString(1)`)\n"
```

Comment:
assuming this indeed works in Clickhouse, this PR looks fine.
Has this been tested?

Reply:
Yes, had test for it. sql like:
select cast('a' as FixedString(1))from system.numbers limit 1;
and result return a.

Reply:
for some reason the CI on windows fails

Reply:
Windows ci error may not related to the changed code of this pr，in my repo test is all passed and also build success in local.

Reply:
![1742538835661.png](https://github.com/user-attachments/assets/d336ea2e-2cd9-40be-9cec-2f41bb8b9192)